Makefile

```diff
@@ -85,9 +106,25 @@ integration-test-with-docker: prepare-docker-runner-image generate-env-file ## R
 		${DOCKER_BUILDER_IMAGE_NAME} \
 		make integration-test
 
+.PHONY: publish-to-pypi-with-docker
+publish-to-pypi-with-docker: prepare-docker-runner-image generate-env-file ## publish wheel to pypi inside a docker container
+	docker run -i --rm \
+		--name "${DOCKER_CONTAINER_PREFIX}-publish-to-pypi" \
+		-v `pwd`:/var/project \
+		-e http_proxy="${HTTP_PROXY}" \
+		-e HTTP_PROXY="${HTTP_PROXY}" \
+		-e https_proxy="${HTTPS_PROXY}" \
+		-e HTTPS_PROXY="${HTTPS_PROXY}" \
+		-e NO_PROXY="${NO_PROXY}" \
+		-e PYPI_USERNAME="${PYPI_USERNAME}" \
```

Comment:
if we would ever pass this in from Jenkins, the values would be still the default ones inside Docker